notifier.yaml

```diff
@@ -9,7 +9,6 @@ handlers:
   # Header checks prevent raw access to this handler.  Tasks have headers.
```

Comment:
Update line 7 to `url: /tasks/email-subscribers`

Reply:
I'm actually purposely keeping that as a catch-all: If any task gets dispatched to the notifier module, then it should be handled by that handler.  If we add more tasks in that module in the future, and if we need another file, then we can add a handler entry at that time.